File: Assignment 41/Example3.c
/* 3. Write a recursive program which dislpay below pattern.
   Input: 5
   Output: 5 4 3 2 1  
*/

#include<stdio.h>

 void Display (int iNo)
 {
    static int iCnt=0;

 if(iNo >= 1)
    {
        printf("%d ", iNo);
        Display(iNo - 1);
    }

 }
int main()
{
    int iValue =0;

    printf("Enter number:");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

File: Assignment 49/Example3.java
/*3.write Java program which accept string from user and display below pattern
 Input: Hello
output:     H  #  #  #  #
            H  e  #  #  # 
            H  e  l  #  #
            H  e  l  l  #
            H  e  l  l  o
*/

import java.util.Scanner;


class Pattern
{
    public void DisplayPattern(String str)
    {
        int iLength = str.length() ;
        int i = 0;
        int j = 0;

        
        for( i = 0; i < iLength; i++)
        {
           
            for( j = 0; j < iLength; j++)
            {
                if(j <= i)    
                {
                    System.out.print(str.charAt(j) + "  ");
                }
                else   
                {
                    System.out.print("#  ");
                }
            }
            System.out.println();
        }

    }
}
class Example3
{
    public static void main(String A []) 
    {
        Scanner sobj = new Scanner(System.in);   
        
        System.out.print("Enter a string: ");

        String str = sobj.nextLine();

        Pattern pobj = new Pattern();
       
        pobj.DisplayPattern(str);
    }
}